compare-coverage.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import sys
+import subprocess
+
+def create_parser():
+    """Command line parser."""
+
+    parser = argparse.ArgumentParser(
+        description='Compare CBMC coverage checking results.'
+    )
+
+    parser.add_argument(
+        'coverage1',
+        help="""
+        JSON coverage output of make-coverage or cbmc-viewer.
+        """
+    )
+    parser.add_argument(
+        'coverage2',
+        help="""
+        JSON coverage output of make-coverage or cbmc-viewer.
+        """
+    )
+    return parser
+
+def dict_diff(dict1, dict2):
+        with open('/tmp/coverage1.json', 'w') as out1:
```

Comment:
Should we ask python for a true tempfile? Hardcoding the names will cause problems if we run more than one process at once.